cpp/oneapi/dal/algo/jaccard/common.cpp

```diff
@@ -22,9 +22,9 @@ namespace jaccard {
 class detail::descriptor_impl : public base {
 public:
     std::int64_t row_begin    = 0;
-    std::int64_t row_end      = 1;
+    std::int64_t row_end      = 0;
     std::int64_t column_begin = 0;
-    std::int64_t column_end   = 1;
+    std::int64_t column_end   = 0;
```

Comment:
minor but applies to the whole lib - should we use column or col? It is related to the observation that ml uses 'coefficients' and graph - 'coeffs'. Asked those questions in the parallel thread

Reply:
column is used in DAL